packages/client/src/client.ts

```diff
@@ -1,3 +1,23 @@
+import * as dotenv from 'dotenv';
+import { Producer } from 'sqs-producer';
+import { validate } from './validator/CustomEmailValidator_client';
+import { v4 as uuidv4 } from 'uuid';
+dotenv.config();
+
 export class NotificationClient {
-  private readonly sqs;
+  private readonly producer = Producer.create({
+    queueUrl: process.env.SQS_QUEUE_URL,
+    region: process.env.AWS_REGION
+  });
+
+  public async sendmessage(msg: object, delaySeconds = 5) {
```

Comment:
```suggestion
  public async sendMessage(msg: object, delaySeconds = 5) {
```
Camel case function name